packages/types/src/options.ts

```diff
@@ -78,7 +78,14 @@ export interface Options {
   /** A global sample rate to apply to all events (0 - 1). */
   sampleRate?: number;
 
-  /** A global sample rate to apply to all transactions (0 - 1). */
+  /**
+   * Sample rate to determine transaction/span sampling.
+   *
+   * 0 = 0% chance of instrumenting
+   * 1 = 100% change of instrumenting
```

Comment:
```suggestion
   * 0.0 = 0% chance of instrumenting
   * 1.0 = 100% chance of instrumenting
```

I would use floats to suggest that `0.5` is a valid choice.
Perhaps even consider noting that `1.0` might be a bad idea... not sure, maybe this is not the place for such advice.